src/types/next-auth.d.ts

```diff
@@ -9,6 +9,7 @@ interface ExtendedUser {
   image?: string;
   email?: string;
   createdAt?: Date;
+  deletedAt?: Date;
```

Comment:
Might be a leftover, I don't think we should be returning the `deletedAt` field for a user session 😛 If the user is soft deleted, then there shouldn't be a session I guess?